int8 msgCnt
int8 rev
FullPositionVector[]  anchorPoint
RTCMheader[] rtcmHeader
RTCMmessageList_t[] msgs
char[] rtcmCharMsg
